packages/scripts/lib/rewire-scripts.js

```diff
@@ -138,7 +154,10 @@ function rewireWebpack(prod) {
         limit: 10000,
         name: `${staticDir}/media/[name].[hash:8].[ext]`,
         // TODO why is this path needed?
-        includePaths: [path.resolve(__dirname, '../../../node_modules/patternfly/dist/fonts/')],
+        includePaths: [
+          path.resolve(paths.appPath, 'node_modules/patternfly/dist/fonts/'),
+          path.resolve(__dirname, '../../../node_modules/patternfly/dist/fonts/'),
```

Comment:
Why have 2 here? Don't they hopefully resolve to the same thing?

Reply:
These are temp measures until we can remove the pattern fly dependency on CSS directly.
We have two levels of node_modules in our monorepo: those at the root and those at the app. The new addition looks in the app whereas before it didn't.